test/lib.categories.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const assert = require('chai').assert;
+const gplay = require('../index');
+
+describe('Categories method', () => {
+  it('should fetch valid list of categories', () => {
+    return gplay.categories().then(categoryIds => {
+      assert.isArray(categoryIds);
+      assert.isTrue(categoryIds.length > 0);
+    });
+  });
+
+  it('should have all categories from constant list of categories', () => {
+    return gplay.categories().then(categoryIds => {
```

Comment:
It should also test the other way around: all categories should have their constant.

Suggestion: use [R.difference](https://ramdajs.com/docs/#difference) to do both checks with a set operation.

Reply:
Hi. What about categories that are not present in dropdown menu? -- APPLICATION and GAME
Maybe add them into resulting array as constants?

Reply:
Yeah, that sounds reasonable

Reply:
Done